index.html

```diff
@@ -940,18 +945,57 @@
 
 ### Rights {#data-rights}
 
-To the extent possible, [=people=] should retain a certain number of rights over their data, and
-these rights should be facilitated by the [=parties=]. While data rights are a self-governing
-mechanism that is not sufficient to support [=privacy=], they do complement self-determination and
-help improve accountability. Such rights should include:
-
-* The right to <em>access</em> [=data=] about oneself.
-* The right to <em>erase</em> [=data=] about oneself, with or without terminating the service (though
-  what [=data=] can be erased may differ between those two cases).
-* The right to <em>port</em> information that one has stored with a given [=party=] in such a way
-  that it can easily be reused or transferred elsewhere.
-* The right to <em>correct</em> [=data=] about oneself, to ensure that one's [=identity=] is
-  properly reflected in a system.
+[=People=] retain certain rights over data about themselves, and these rights should be facilitated
+by their [=user agent=] and the [=parties=] they're interacting with. While data rights alone are
+not sufficient to satisfy all [=privacy=] principles for the Web, they do support self-determination
+and help improve accountability. Such rights include:
+
+* The <dfn data-export="">right to access</dfn> [=data=] about oneself.
+
+This right includes being able to discover data about oneself (implying no databases are kept
+secret) and to review
+what information has been collected or inferred.
+
+* The <dfn data-export="" data-lt="right to be forgotten">right to erase</dfn> [=data=] about
+oneself, sometimes described as a <em>right to be forgotten</em>.
```

Comment:
The terminology "right to be forgotten" has become enmeshed with search engine erasure and the such. It's a fascinating area, but I wonder if we should be careful about stepping into it?

Reply:
Per group conversation today, it seems like the nuanced and sometimes confusing colloquial term "right to be forgotten" might not be the most useful here, so it could be removed. The subtle questions about deleting data about oneself will still exist, but don't need to be resolved in this section. I'll remove this clause, and then merge this PR.